src/pages/profileScreen.vue

```diff
@@ -98,58 +99,66 @@ function handleShowMFAProceedModal(show: boolean) {
 }
 
 async function handleMFASetupClick() {
-  cleanExit = false
-  const mfaSetupPath = new URL(`mfa/${appStore.id}/setup`, AUTH_URL)
-  if (appStore.standaloneMode == 0) {
-    mfaWindow = window.open(
-      mfaSetupPath.toString(),
-      '_blank',
-      getWindowFeatures()
-    )
+  const { loginType } = JSON.parse(
+    getStorage().session.getItem('userInfo') as string
```

Comment:
It looks like we are relying on local storage to pass state around, don't we have a better way to do this?

Reply:
I guess we can add this loginType to the store as well. But it wasn't fitting any store like app or user. Anyway we have these info in sessionStorage to persist login in a single tab, so I thought we could get this info from here. I'll add it to user store